lib/ui/settings/about_section_widget.dart

```diff
@@ -0,0 +1,130 @@
+// @dart=2.9
+
+import 'package:flutter/material.dart';
+import 'package:photos/services/update_service.dart';
+import 'package:photos/ui/common/web_page.dart';
+import 'package:photos/ui/components/captioned_text_widget.dart';
+import 'package:photos/ui/components/expandable_menu_item_widget.dart';
+import 'package:photos/ui/components/menu_item_widget.dart';
+import 'package:photos/ui/settings/app_update_dialog.dart';
+import 'package:photos/ui/settings/common_settings.dart';
+import 'package:photos/utils/dialog_util.dart';
+import 'package:photos/utils/toast_util.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class AboutSectionWidget extends StatelessWidget {
+  const AboutSectionWidget({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ExpandableMenuItemWidget(
+      title: "About",
+      selectionOptionsWidget: _getSectionOptions(context),
+      leadingIcon: Icons.info_outline,
+    );
+  }
+
+  Widget _getSectionOptions(BuildContext context) {
+    return Column(
+      children: [
+        sectionOptionSpacing,
+        const AboutMenuItemWidget(
+          title: "FAQ",
+          webPageTitle: "FAQ",
+          url: "https://ente.io/faq",
+        ),
+        sectionOptionSpacing,
+        const AboutMenuItemWidget(
+          title: "Terms",
+          webPageTitle: "terms",
+          url: "https://ente.io/terms",
+        ),
+        sectionOptionSpacing,
+        const AboutMenuItemWidget(
+          title: "Privacy",
+          webPageTitle: "privacy",
```

Comment:
Now that text casing has changed on web too, we can use "Privacy" and "Terms" as title.

Reply:
After that, may be we can make `webPageTitle` as optional param. We can use `title` if `webPageTitle` is missing